src/com/epam/javatrainings/annotation/validator/Validator.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.javatrainings.annotation.validator;
+
+import com.epam.javatrainings.annotation.model.CustomerDto;
+
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Validator {
+  private final Map<String, String> error;
```

Comment:
#76 
You declared it as an instance variable instead of local(in method body)

Reply:
is fixed